guides/micronaut-oracle-function/java/src/main/java/example/micronaut/Function.java

```diff
@@ -0,0 +1,22 @@
+package example.micronaut;
+
+import io.micronaut.core.annotation.ReflectiveAccess;
+import io.micronaut.oraclecloud.core.TenancyIdProvider;
+import io.micronaut.oraclecloud.function.OciFunction;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.util.Date;
+
+@Singleton
+public class Function extends OciFunction {
+
+    @Inject
+    TenancyIdProvider tenantIdProvider;
+
+    @ReflectiveAccess
+    public String handleRequest() {
```

Comment:
Why do we need `@ReflectiveAccess` here?

Reply:
Because Project.fn invokes the function handler via reflection, which is beyond our control

Reply:
I though that the annotation was only necessary for GraalVM, but the guide doesn't use GraalVM.
In any case I think adding an explanation in the guide about why is necessary is good for users.

Reply:
The guide needs to be updated to cover native image as well